### DISCUSSION TITLE
做讀書計畫的訣竅

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1366] [General]
大家做讀書計畫時，都有按步就班、還是常常半途而廢呢? 計畫並不是越多越好，要怎麼樣才能持續執行呢?
第一步:設定目標=>規劃前要先確認目標，目標不是越大越好，以能執行及明確為主
第二步:列出範圍並評估時間=> 列出詳細的讀書範圍，有哪些教材要讀
第三步:找出可運用時間=>扣除平日的固定行程，排出可運用及零碎的時間
第四步:將進度填入讀書計畫表
##別忘了留一些休閒跟運動的時間!!

**Comment by llh26** [comment]
我個人喜歡利用「列點式」去設立目標，這樣可以超清楚知道自己有什麼事情還沒做，在執行事情上也可以更有效率，也會幫每件事情都設立deadline，可以有效改善拖延症的問題，甚至還可以超前進度完成任務，在其他時間的安排上也須注意（娛樂、休息⋯），一定要留時間讓自己處理事情，不然會變成拖延症大師⋯。

**Comment by alicia0725** [comment]
我個人會盡量把目標設小，執行起來會相對有動力一些，當小目標逐漸累計，也更容易獲得成就感。除此之外，我也發現要盡量避免一天排定過多的目標，這樣才有足夠的休息，讓第二天還有動力去執行計畫。

**Comment by Alexgns** [comment]
了解自己的學習風格和偏好，並據此調整讀書計畫。有些人喜歡閱讀，而其他人可能更適合聆聽錄音或參加討論，在達到重要的里程碑或完成困難的任務後，給自己一些小獎勵。可以提供額外的動力和動力，能夠讓自己保持積極性

**Comment by ALAN30359704** [comment]
事實上我並不是一個非常會做讀書計劃的人，因為常常我無法照著計劃做事情，但是我覺得我的讀書方法是平常照著學校的進度按部就班地唸書，不要把工作或作業累積到最後在教平時就慢慢消化跟唸書就不會到最後要擬定非常緊迫的工作量去完成！

**Comment by Elaine2899** [comment]
進入大學之後我開始嘗試做讀書計畫，一開始我像高中一樣手足無措，經常是做了計劃但又無法履行，次次讓自己失望懊惱。經過幾次的失敗與檢討之後，我慢慢開始可以依照自己的計畫執行。我使用的方法是先決定大目標，然後細分成小小的短期的目標，再列出一個一個的小項目，將這些小項目列入我的代辦清單，當按下代辦清單的"完成"的方框，並將這個代辦項目刪除的時候會讓我特別有成就感。雖然還沒有辦法大半都依照自己的計劃行動，但這樣的改變已經讓我很滿足了，也相信隨著每一次的失敗與檢討，我會更加習慣跟著計劃行動，並規劃適合自己的計畫與目標。

**Comment by alstinlin** [comment]
在定讀書計畫時我習慣定一個長期的目標，比如說在一到兩個月把某個科目複習一次或把筆記整理好，因為我喜歡時間比較彈性，也不太會被臨時的狀況影響，但相對的也需要我每天有規律的實行，否則容易在時間快到的時候才發現進度落後

**Comment by sjh501** [comment]
我習慣先有個大方向，當執行時再依據當時的情況做調整，譬如最近準備段考，先確定考試範圍，計畫每天大概進度多少，在執行的當下在根據實際可以運用的時間跟身體狀況調整。我個人讀書的狀況很吃當下的身體狀況，如果很累或者不適不會勉強自己要達成，但狀況好的時候就會提醒自己今天要有更多進度。

**Comment by juliaouo** [comment]
做讀書計畫的時候通常都想得很美，覺得自己一定可以在什麼時候做完什麼事，但可能是我自己個性的原因，通常不到死線很難把事情撿起來做，就算有制定讀書激化也很難遵照著執行，每次都想改，但是江山易改，本性難移XD

**Comment by Aaron0810** [comment]
我認為的讀書計畫有兩種，這兩種個有優缺，一是安排較少的事情，但要確保當天做的完，不過只要做完了不用去想要多做，二是安排多羿點的計畫，安排過多看起來會比較擁擠，會讓你念得比較有壓力但速度會比較快，可能把本來兩天份的作業安排在一天，雖然一天真的做不完兩天份但有機會做完1.5天的份，也算是一種賺到，這兩種我都會用，但也說不上哪種比較好。

**Comment by ashleyy-huang** [comment]
我會把行程列出來，每段時間該讀什麼該做什麼詳細的一項一項條列出來，如果有沒讀完的，我便會開始審視我自己是不是讀書效率沒有我自己預期的好，然後盡可能的把沒完成的進度在當天補齊，全部都讀完後，我也會思考今天這樣排的進度對我來說會不會太輕鬆又或是排太滿，之後就所調整。

**Comment by lyien1021** [comment]
我是一個蠻喜歡寫讀書計畫的人，不過常常沒辦法達到我所有制定的目標，除了我有拖延症之外，我想做的事情又太多，導致我總是完成不了，因此我現在都只定大方向的目標了，不然總是達不到也是會蠻沮喪。

**Comment by KozakHou** [comment]
我這個人從來不做讀書計畫。 當有很多事要做的時候，我會全部列在清單上，然後，開始做 !
反正都要做完，所以順序就感覺不是這麼重要了。
這也是為什麼大家越期末越忙，我越期末越輕鬆

就我個人觀點來看，我會一直奉此為圭臬、一生遵循這種方式，將我能做的事，全部做完。事情做完，再找事做，周而復始，生生不息~

**Comment by JessyCheng2003** [comment]
我的方法是將當天的代辦事項寫在本子上，完成的話就劃掉，這樣可以盤點我每天做了什麼事情、完成什麼作業，也會讓我有一點點成就感。

**Comment by pudding0803** [comment]
我平常比較不會有長期的計劃，因為根據我之前的經驗，只要有一天沒有符合到進度，我就會有點懶得繼續完成計畫😢，所以目前都比較沒有計畫的習慣😅。但 To-do List 可能是個很好的選擇！如果自己會管理時間做事情的話，就可以有效提醒自己近期該做的事，時間也都可以自己安排。

**Comment by Superlan8** [comment]
我不喜歡制定那種太細的讀書計畫，我喜歡訂一個大方向跟日期，只要在那個目標前完成就好，因為每天情況都不同，沒辦法去預測每一天的情況，而且還要視當天讀書效率決定，太細反而會影響我的書時的節奏。

**Comment by Brownnnn78** [comment]
我覺得除了規劃很重要之外，最重要的是惰性，把該做的事列出來總是發現時間不夠，感覺每項作業都很難因此越拖越晚，但實際上如果先花5分鐘了解整項作業或任務要做哪些事，花一點時間就能夠清楚接下來該做什麼，而且有時最難的就是開始的時候，通常我只要開始了就會想趕快完成！
